src/configure_settings/configure_settings.jl

```diff
@@ -214,3 +237,162 @@ function validate_multistage_settings!(settings::Dict{Any, Any})
         settings["WriteIntermittentOutputs"] = 0
     end
 end
+
+function default_input_names(case::AbstractString)
+    Dict{Any, Any}("system_location" => joinpath(case, "system"),
+    "demand" => "Demand_data.csv",
+    "fuel" => "Fuels_data.csv",
+    "generators" => "Generators_variability.csv",
+    "network" => "Network.csv",
+    "resources_location" => joinpath(case, "resources"),
+    "storage" => "Storage.csv",
+    "thermal" => "Thermal.csv",
+    "vre" => "Vre.csv",
+    "vre_stor" => "Vre_stor.csv",
+    "vre_stor_solar_variability" => "Vre_and_stor_solar_variability.csv",
+    "vre_stor_wind_variability" => "Vre_and_stor_wind_variability.csv",
+    "hydro" => "Hydro.csv",
+    "flex_demand" => "Flex_demand.csv",
+    "must_run" => "Must_run.csv",
+    "electrolyzer" => "Electrolyzer.csv",
+    "resource_cap" => "Resource_capacity_reserve_margin.csv",
+    "resource_energy_share_requirement" => "Resource_energy_share_requirement.csv",
+    "resource_min" => "Resource_minimum_capacity_requirement.csv",
+    "resource_max" => "Resource_maximum_capacity_requirement.csv",
+    "resource_hydrogen_demand" => "Resource_hydrogen_demand.csv",
+    "resource_hourly_matching" => "Resource_hourly_matching.csv",
+    "resource_multistage_data" => "Resource_multistage_data.csv",
+    "policies_location" => joinpath(case, "policies"),
+    "period_map" => "Period_map.csv",
+    "capacity" => "Capacity_reserve_margin.csv",
+    "CRM_slack" => "Capacity_reserve_margin_slack.csv",
+    "co2_cap" => "CO2_cap.csv",
+    "co2_cap_slack" => "CO2_cap_slack.csv",
+    "esr" => "Energy_share_requirement.csv",
+    "esr_slack" => "Energy_share_requirement_slack.csv",
+    "min_cap" => "Minimum_capacity_requirement.csv",
+    "max_cap" => "Maximum_capacity_requirement.csv",
+    "operational_reserves" => "Operational_reserves.csv")
```

Comment:
It might make sense to have this be a `dict{Symbol, String}` or `dict{Symbol, Any}`. i.e. the keys of the dict should be Symbols, not strings. In general symbols are nicer as identifiers where you don't need to decompose the strings into characters or concatenate them into longer strings.